src/main/java/com/iexec/sms/utils/EthereumCredentials.java

```diff
@@ -45,21 +47,31 @@
 
     private String privateKey;
     private boolean isEncrypted;
+    /*
+     * Address is required since recovering from private key is not possible
+     * from an encrypted private key state.
```

Comment:
Is it to enable using this object without decrypting the private key?

Reply:
Yes exactly